### PR TITLE
Bug: Patch Failing Dockerfile

### LINKED ISSUE #83: Bug: Patch Failing Docker Image
The Dockerfile needs to be updated to include this patch:
https://github.com/xmidt-org/tr1d1um/commit/df49bb536fa91ca46fec6384534ddc674406a2b9

## Steps:
- [ ]  create missing entrypoint.sh
- [ ]  fix breaking chrysom changes
- [ ]  fix breaking argus changes
- [ ]  fix breaking wrp-listener changes
- [ ]  fix linter errors
- [ ]  replace deprecated library usage of `Maintainer`

### PR DESCRIPTION
**Pull request by denopink**
## What's included:
- Patch failing build layers
- Remove deprecated `MAINTAINER`
- Fixed linter issues and enabled linter

Closes #83 

https://github.com/comcast-cl/xmidt/issues/446
https://github.com/comcast-cl/xmidt/issues/435

### CODE REVIEW

**Review by johnabass — APPROVED** [review]

File: registry/registry.go [hunk 1]
```diff
@@ -52,13 +55,11 @@ func jsonResponse(rw http.ResponseWriter, code int, msg string) {
...
 
-	argus, err := chrysom.CreateClient(in.NornRegistry.Argus, chrysom.WithLogger(in.NornRegistry.Logger))
+	argus, err := chrysom.NewBasicClient(in.NornRegistry.Argus, nil)
 	if err != nil {
 		return nil, err
 	}
-	if in.NornRegistry.Listener != nil {
-		argus.SetListener(in.NornRegistry.Listener)
-	}
+
```
Comment (renaz6) [inline registry/registry.go:62]: I removed `argus.SetListener(in.NornRegistry.Listener)`. The Argus/Chrysom package was changed last year to split up the Chrysom client into a BasicClient and a Listener client (https://github.com/xmidt-org/argus/issues/147). Therefore, this if statement and `SetListener()` call were no longer applicable. 

Instead, should this function also include a call to `chrysom.NewListenerClient()`? https://github.com/xmidt-org/argus/blob/7a4557764548b354c095b3df115eb57f5fbe119f/chrysom/listenerClient.go#L88

File: main.go [hunk 2]
```diff
@@ -160,7 +161,7 @@ func main() {
 			func(v *viper.Viper, m *manager.Manager) (registry.NornRegistry, error) {
 				config := new(registry.NornRegistry)
 				err := v.UnmarshalKey("nornRegistry", &config)
-				config.Listener = m.Update
+				config.Listener = m.Update //nolint: typecheck
```
[the remark this answers is not in the document]
Comment (schmidtw) [reply main.go]: It looks like this change happened when we moved from argus to chrysom.  This was probably very early days for argus & has been broken for a long time.

File: registry/registry.go (hunk 1 above)
Comment (schmidtw) [reply registry/registry.go:62]: I think the functionality moved from Argus to chrysom & we should favor chrysom APIs over argus.

Comment (renaz6) [reply registry/registry.go:62]: Thanks for helping me look into this!

**Review by schmidtw — APPROVED** [review]

**Pull request merged**